<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="processState">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>9</y>
               <width>253</width>
               <height>18</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
              </font>
             </property>
             <property name="text">
              <string>Состояние процесса: Выключен</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="isDebugMode">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>33</y>
               <width>151</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Режим отладки </string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="killProcessBtn">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>DejaVu Sans</family>
             </font>
            </property>
            <property name="text">
             <string>Убить процесс</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="logText">
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>11</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>540</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>540</height>
        </size>
       </property>
       <widget class="QPushButton" name="testAllBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Запустить все тесты</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testUartBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест UART</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testRtcBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>350</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест RTC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testHdmiBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест HDMI</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testPciBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест PCI</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testRs422Btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест RS422</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testUsbBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест USB</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testCanBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест CAN</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testEthBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>500</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест Ethernet</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testEmmcBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>450</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест EMMC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testSataBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Тест SATA</string>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>190</y>
          <width>16</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="testUartEndlessBtn">
        <property name="geometry">
         <rect>
          <x>159</x>
          <y>200</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>∞</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
